A2rchi/interfaces/chat_app/app.py

```diff
@@ -177,14 +296,26 @@ def __call__(self, history: Optional[List[Tuple[str, str]]], discussion_id: Opti
             else:
                 output = "<p>" + self.format_code_in_text(result["answer"]) + "</p>"
 
-            ChatWrapper.update_or_add_discussion(self.data_path, "conversations_test.json", discussion_id, discussion_contents = history + [("A2rchi", output)])
+            # write user message and A2rchi response to database
+            user_message = (sender, content, msg_ts)
+            a2rchi_message = ("A2rchi", output, datetime.now())
+            print(user_message)
```

Comment:
I agree with @ludomori99 that print statements such as these need at least minimal context in them (this just needs to be quick and dirty, we can swap to more robust logging later if needed).